coffeecard/CoffeeCard.WebApi/wwwroot/Templates/EmailTemplate/email_verify_registration.html

```diff
@@ -1,356 +1,173 @@
-                                                                        </tbody>
-                                                                    </table>
-                                                                </div>
-                                                            </td>
-                                                        </tr>
-                                                        </tbody>
-                                                    </table>
-                                                </td>
-                                            </tr>
-                                            </tbody>
-                                        </table>
-                                    </div><!--[if (gte mso 9)|(IE)]></td><![endif]--><!--[if (gte mso 9)|(IE)]></td></tr></table><![endif]-->
+                    <td style="border-collapse: collapse !important; margin: 0 auto; mso-table-lspace: 0 !important; mso-table-rspace: 0 !important;cellpadding: 0; cellspacing: 0; border: 0;">
+                        <h1 class="wrapperH1" style="text-align: center; font-weight: 700; line-height: 1.5em; Margin: 0; font-size: 1.4em; padding: 0px !important;">
+                            Hello {name},
+                        </h1>
+                    </td>
+                </tr>
+                <tr>
+                    <td style="border-collapse: collapse !important; margin: 0 auto; mso-table-lspace: 0 !important; mso-table-rspace: 0 !important;cellpadding: 0; cellspacing: 0; border: 0; align-content: center">
+                        <table border="0" cellpadding="0" cellspacing="0" align="center" width="90%" style="font-family: 'Nunito Sans', 'Helvetica Neue', helvetica, arial, sans-serif;  color:#38251a; align-content: center; text-align: center; margin: 0 auto; border-collapse: collapse !important; mso-table-lspace: 0 !important; mso-table-rspace: 0 !important;cellpadding: 0; cellspacing: 0; border: 0;">
+                            <tr>
+                                <td style="border-collapse: collapse !important; margin: 0 auto; mso-table-lspace: 0 !important; mso-table-rspace: 0 !important;cellpadding: 0; cellspacing: 0; border: 0; font-size: 1em">
+                                    <p style="margin: 0;>
+                                        You are almost there, you just need to verify that {email} is the email you want to use within the next {expiry}.
+                                    </p>
+                                    <!-- ======= start spacer ======= -->
+                                    <div style="font-size: 1px; height: 20px; line-height: 20px; Margin: 0; mso-line-height-rule: exactly;">&nbsp;</div>
+                                    <!-- ======= end spacer ======= -->
+                                    <p style="margin: 0;">
+                                        If you did not create an account with us you can safely disregard this message.
```

Comment:
I think some welcome text is missing, could you add:
```suggestion
Thanks for registrering with the Cafe Analog App.

We need to verify your email before you can login. Click the button below to verify your email.

If you did not create an account with us, you can disregard this message.
```

Reply:
Maybe the user should contact us if someone registres with their email? Else it would cause trouble in the future. To be discussed

Reply:
There was already the following as an intro message in the above paragraph: "_You are almost there, you just need to verify that {email} is the email you want to use within the next {expiry}._"

Your message might be a bit nicer, so I'll change the message to that. I don't think it would be a good idea to rely on the user to message us if they did not request the account, so if we are concerned about people using other emails to try and sign up I think we should take a different approach. I think it should be possible to do it using tokens so nothing is added to the database, except a token, before the email is validated.

Reply:
That sounds like a good approach, could you create a small ticket on this on Trello?